model/project_ref.go

```diff
@@ -1867,15 +1900,21 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 		if p.TracksPushEvents != nil {
 			setUpdate[ProjectRefTracksPushEventsKey] = p.TracksPushEvents
 		}
-		// Allow a user to modify owner and repo only if they are editing an unattached project
 		if !isRepo && !p.UseRepoSettings() && !defaultToRepo {
-			allowedOrgs := evergreen.GetEnvironment().Settings().GithubOrgs
-			if err := p.ValidateOwnerAndRepo(allowedOrgs); err != nil {
+			config := evergreen.GetEnvironment().Settings()
+			// Allow a user to modify owner and repo only if they are editing an unattached project
+			if err := p.ValidateOwnerAndRepo(config.GithubOrgs); err != nil {
 				return false, errors.Wrap(err, "validating new owner/repo")
 			}
 
 			setUpdate[ProjectRefOwnerKey] = p.Owner
 			setUpdate[ProjectRefRepoKey] = p.Repo
+
+			// Cannot enable projects if the project creation limits have been reached.
+			err, _ = ValidateProjectCreation(projectId, config, p)
```

Comment:
Why is this not doing anything with the `shouldError` bool?

Reply:
I still curious about why the shouldError field is not used here to decide if we should error on line 1910?

Reply:
ah its because should error doesnt matter if there is no error 
the boolean just tells us what we should do with the error (ie if it's false we will warn them instead of erroring)

Reply:
At this point we don't know if there is an error or not because we are checking it on line 1910. What if there's an error and shouldError is false, should we not change the behavior on 1911 to not return the error?

Reply:
ah sorry, I didn't look closely enough. 
I added a check and a todo so we can warn users once that ticket is done